pkgs/by-name/he/hexpatch/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  cmake,
+  python3,
+  stdenv,
+  gcc,
+  llvmPackages,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hexpatch";
+  version = "1.9.3";
+
+  src = fetchFromGitHub {
+    owner = "Etto48";
+    repo = "HexPatch";
+    rev = "v${version}";
+    hash = "sha256-yyYxRJ+o+Z5z7PmjcFCsahRXZ9JHFmGmituzGTxY6ec=";
+  };
+
+  cargoHash = "sha256-bckZtHzgKQow7l2Cg6Xi1oNk13FH2xKe8cbZ/Xj9E04=";
```

Comment:
These hashes need to be updated. I assume this built fine for you. That usually happens because Nix doesn't actually verify the hashes again if they are already cached locally. You can force a refetch by setting these hashes to an empty string.

Nix should then report the new (correct) hashes.